### PR TITLE
chore: bump dependencies (#2385 - #2396)

### PR DESCRIPTION
**Pull request by mykeels**
**Overall change:** _Bumping third party dependencies._

**Code changes:**

- Covers the following dependabot PRs
#2396, #2395, #2394, #2393, #2392, #2391, #2390, #2389, #2388, #2387, #2386, #2385

---

- [x] I have assigned myself to this PR
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires regression testing

### CODE REVIEW

**Review by FK78 — APPROVED** [review]
LGTM!

**Review by hotinglok — APPROVED** [review]

**Review by thekp — APPROVED** [review]
💯

**Review by thekp — DISMISSED** [review]
https://github.com/bbc/psammead/pull/2400/files#diff-32607347f8126e6534ebc7ebaec4853dR3

this needs updating with an `npm install` or `npm run ci:packages`

**Review by FK78 — APPROVED** [review]

**Review by hotinglok — APPROVED** [review]

**Review by EinsteinNjoroge — APPROVED** [review]

**Comment by chris-hinds** [comment]
CI is failing as 

```
Branch must update CHANGELOG.md in psammead-timestamp-container
Branch must update package-lock.json in psammead-timestamp-container
Branch must update package.json in psammead-timestamp-container
```

need to be updated in psammead-timestamp-container

File: packages/containers/psammead-timestamp-container/CHANGELOG.md [hunk 1]
```diff
@@ -43,12 +44,12 @@
...
-| 1.2.6   | [PR#1083](https://github.com/bbc/psammead/pull/1083) use `moment-timezone@0.5.26` |
-| 1.2.5   | [PR#1038](https://github.com/bbc/psammead/pull/1038) Bump dependencies |
-| 1.2.4 | [PR#892](https://github.com/bbc/psammead/pull/892) Use `@bbc/psammead-timestamp@1.0.3` |
-| 1.2.3 | [PR#783](https://github.com/bbc/psammead/pull/783) Update to latest psammead-test-helpers. Update snapshots. |
-| 1.2.2 | [PR#677](https://github.com/bbc/psammead/pull/677) Use `@bbc/gel-foundations@3.0.0` |
-| 1.2.1 | [PR#666](https://github.com/bbc/psammead/pull/666) Do not localise `datetime` |
-| 1.2.0 | [PR#643](https://github.com/bbc/psammead/pull/643) Add support for passing locales as prop |
-| 1.1.0 | [PR#592](https://github.com/bbc/psammead/pull/592) Receive a padding; pass a padding |
-| 1.0.0 | [PR#532](https://github.com/bbc/psammead/pull/532) Initial creation of package |
+| 1.2.6   | [PR#1083](https://github.com/bbc/psammead/pull/1083) use `moment-timezone@0.5.26`                                                                                  |
```
Comment (chris-hinds) [inline packages/containers/psammead-timestamp-container/CHANGELOG.md]: whats all the extra whitespace?

**Review by chris-hinds — APPROVED** [review]
LGTM

File: packages/containers/psammead-timestamp-container/src/__snapshots__/index.test.jsx.snap [hunk 2]
```diff
@@ -114,4 +114,4 @@ exports[`Timestamp should render without a leading zero on the day 1`] = `
 >
   ۷ ژوئیه ۲۰۱۸
 </time>
-`;
+`;
```
Comment (jroebu14) [inline packages/containers/psammead-timestamp-container/src/__snapshots__/index.test.jsx.snap]: what happened here?

**Review by jroebu14 — APPROVED** [review]

File: packages/containers/psammead-timestamp-container/src/__snapshots__/index.test.jsx.snap (hunk 2 above)
Comment (FK78) [reply packages/containers/psammead-timestamp-container/src/__snapshots__/index.test.jsx.snap]: EOL removed by mistake, fixed.

**Pull request merged**